std_execution.bs

```diff
@@ -2853,33 +2854,33 @@ namespace std::execution {
   template &lt;class T, class... An>
     concept receiver_of = <i>see-below</i>;
 
-  inline namespace <i>unspecified</i> {
+  namespace <i>unspecified</i> {
     struct set_value_t;
-    inline constexpr set_value_t set_value{};
     struct set_error_t;
-    inline constexpr set_error_t set_error{};
     struct set_done_t;
-    inline constexpr set_done_t set_done{};
   }
+  inline constexpr <i>unspecified</i>::set_value_t set_value{};
+  inline constexpr <i>unspecified</i>::set_error_t set_error{};
+  inline constexpr <i>unspecified</i>::set_done_t set_done{};
 
   // [execution.receivers.queries], receiver queries
-  inline namespace <i>unspecified</i> {
+  namespace <i>unspecified</i> {
     struct forwarding_receiver_query_t;
-    inline constexpr forwarding_receiver_query_t forwarding_receiver_query{};
 
     template &lt;class T>
       concept <i>forwarding-receiver-query</i> = // exposition only
-        forwarding_receiver_query(T{});
+        forwarding_receiver_query_t{}(T{});
   }
```

Comment:
Do we need to specify this concept? Is implementation detail.

Reply:
Leave it for now. There's been some drift. When `get_scheduler` and friends became "general" queries, `forwarding_receiver_query` didn't come along for the ride. I'll file an issue.